packages/plugin-user/test/integration/spec/user.js

```diff
@@ -0,0 +1,104 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import '../..';
+
+import {assert} from '@ciscospark/test-helper-chai';
+import sinon from '@ciscospark/test-helper-sinon';
+import CiscoSpark from '@ciscospark/spark-core';
+import testUsers from '@ciscospark/test-helper-test-users';
+import {patterns} from '@ciscospark/common';
+import uuid from 'uuid';
+
+describe(`plugin-user`, function() {
+  this.timeout(10000);
+
+  let spark, user2, user3;
+
+  before(() => testUsers.create({count: 3})
+    .then((users) => {
+      const user = users[0];
+      user2 = users[1];
+      user3 = users[2];
+      spark = new CiscoSpark({
+        credentials: {
+          authorization: user.token
+        }
+      });
+      assert.isTrue(spark.isAuthenticated);
+
+      return spark.device.register();
+    }));
+
+  describe(`#get()`, () => {
+    it(`gets the current user`, () => spark.user.get()
+      .then((user) => {
+        assert.equal(user.id, spark.device.userId);
+        assert.property(user, `entitlements`);
+        assert.property(user, `email`);
+        assert.property(user, `name`);
+      }));
+  });
+
+  describe(`#asUUID()`, () => {
+    function makeEmailAddress() {
+      return `spark-js-sdk--test-${uuid.v4()}@example.com`;
+    }
+
+    let email;
+    beforeEach(() => {
+      email = makeEmailAddress();
+    });
+
+    it(`maps an email address to a uuid`, () => assert.eventually.equal(spark.user.asUUID(user2, {force: true}), user2.id));
+
+    it(`maps an email address for a non-existent user to a fake uuid`, () => assert.eventually.match(spark.user.asUUID(email), patterns.uuid)
```

Comment:
I see now we are checking that there is an added entry in the store that has the userExists=false flag